v23.1/fips-compliance.md

```diff
@@ -0,0 +1,355 @@
+{{site.data.alerts.end}}
+
+A system must have FIPS mode enabled in the kernel before it can run the FIPS-ready CockroachDB binary or that starts containers that run the FIPS-ready CockroachDB Docker image. Enabling FIPS mode loads the FIPS kernel module so that the kernel itself enforces compliance with FIPS 140-2. To enable FIPS mode on Red Hat Enterprise Linux or its derivatives, refer to [Enable FIPS mode](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening) in Red Hat's documentation. To verify that FIPS mode is enabled, refer to [Verify that the kernel enforces FIPS mode](#verify-that-the-kernel-enforces-fips-mode).
+
+### Extend Red Hat's Universal Base Image 8 Docker image
+
+If you do not want to use the FIPS-ready CockroachDB Docker image directly, you can create a custom Docker image based on [Red Hat's Universal Base Image 8 Docker image](https://catalog.redhat.com/software/containers/ubi8/ubi/).
+
+- You can model your Dockerfile on the one that Cockroach Labs uses to produce the [FIPS-ready Docker image](https://github.com/cockroachdb/cockroach/blob/master/build/deploy/Dockerfile) for CockroachDB.
+- Your Dockerfile must install OpenSSL before it starts the `cockroach` binary.
+- FIPS mode must be enabled on the Docker host kernel before it can run containers with FIPS mode enabled. The FIPS-ready CockroachDB Docker image must run with FIPS mode enabled. To enable FIPS mode in the Docker host kernel, refer to [Enable FIPS mode](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening) in Red Hat's documentation. To verify that FIPS mode is enabled, refer to [Verify that the kernel enforces FIPS mode](#verify-that-the-kernel-enforces-fips-mode).
+
+### Install on Red Hat Enterprise Linux
+
+When you install Red Hat Enterprise Linux (RHEL) on a host, or after installation has completed, you can [enable FIPS mode](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening). Red Hat recommends installing RHEL with FIPS mode enabled, as opposed to enabling FIPS mode later to help ensure that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
+
+After FIPS mode is enabled, when you install OpenSSL, FIPS-validated OpenSSL libraries are installed. To install OpenSSL on RHEL:
```

Comment:
This seems a bit more straightforward

```suggestion
Once FIPS mode is enabled, install OpenSSL so that the FIPS-validated OpenSSL libraries are installed. 

To install OpenSSL on RHEL:
```